packages/shared/src/components/plus/PlusComparingCards.tsx

```diff
@@ -90,10 +92,18 @@ const PlusCard = ({
         </Typography>
         {plan?.extraLabel && (
           <PlusPlanExtraLabel
-            color={PlusLabelColor.Help}
-            label={plan.extraLabel}
+            color={
+              isFreeTrialExperiment
+                ? PlusLabelColor.Success
+                : PlusLabelColor.Help
+            }
+            label={isFreeTrialExperiment ? '7-day free trial' : plan.extraLabel}
```

Comment:
Why don't we just re-use the extraLabel for this plan?

Reply:
We have different label shown in onboarding and plus page, so I've used the extraLabel on checkout and a static one during onboarding. Maybe is an overkill to add a new customData field only for an experiment, wdyt?

Reply:
Ah I see, 2 different ones are a pain.
Just verify product is ok with it, they might expect to be able to control the copy?